src/session.rs

```diff
@@ -59,22 +305,28 @@ impl Client {
             ack: tx,
         });
 
-        async move {
-            if r.is_err() {
-                return Err(error::CmdError::Lost(io::Error::new(
-                    io::ErrorKind::BrokenPipe,
-                    "WebDriver session has been closed",
-                )));
-            }
-
-            let r = rx.await;
-            r.unwrap_or_else(|_| {
-                Err(error::CmdError::Lost(io::Error::new(
-                    io::ErrorKind::BrokenPipe,
-                    "WebDriver session was closed while waiting",
-                )))
-            })
+        if r.is_err() {
+            return Err(error::CmdError::Lost(io::Error::new(
+                io::ErrorKind::BrokenPipe,
+                "WebDriver session has been closed",
+            )));
         }
+
+        let r = rx.await;
+        r.unwrap_or_else(|_| {
+            Err(error::CmdError::Lost(io::Error::new(
+                io::ErrorKind::BrokenPipe,
+                "WebDriver session was closed while waiting",
+            )))
+        })
+    }
+
+    /// Issue the specified [`WebDriverCompatibleCommand`] to the WebDriver instance.
+    pub async fn issue_cmd(
+        &self,
+        cmd: Box<dyn WebDriverCompatibleCommand + Send>,
```

Comment:
Should this take `impl Into<Box<dyn WebDriverCompatibleCommand + Send>>`? Or maybe just `C: WebDriverCompatibleCommand + Send`, and then we `Box` it internally?

Reply:
that's a good idea. Will do.